app/models/rdv.rb

```diff
@@ -34,6 +34,9 @@ class Rdv < ApplicationRecord
   after_save :associate_users_with_organisation
   after_update :send_notifications_to_users, if: -> { saved_change_to_starts_at? && notify? }
 
+  after_save :send_web_hook
+  before_destroy :send_web_hook_on_destroy, prepend: true
```

Comment:
why not use `after_destroy` ? the instance would then have the 'right' status and you wouldn't have to override it manually I think